scripts/actor/actor-aos.js

```diff
@@ -349,8 +349,9 @@ export class AgeOfSigmarActor extends Actor {
      */
     async applyDamage(damage, {ignoreArmour = false, penetrating = 0, ineffective = false, restraining = false}={}) {
         let armour = this.combat.armour.value
-
-        armour -= penetrating
+        
+        if (armour > 0)
+            armour -= penetrating
```

Comment:
Wouldn't this still sometimes cause penetrating to deal extra damage? e.g. 1 Armour, with 2 Penetrating = 1 extra damage

Reply:
I wouldn't the penetraiting trait ignores 1 point of armour not more not less. A change is only neccessary if we were to use that value to implement another effect